minos/networks/discovery/clients/minos_discovery.py

```diff
@@ -0,0 +1,53 @@
+"""minos.networks.discovery.clients module."""
+
```

Comment:
Only a small detail, I think it's preferable to name the file simply as `minos.py` instead of `minos_discovery.py` for sake or simplicity. I know that it's the same name of the framework, but as this file is located in a relative path, there should no be any problem with that. 😉

Reply:
I thought of that as well, but didn't like to have a `minos.py` file :sweat_smile:. It's true that the namespace should do the job, though, so I'll change it.

Reply:
Done.